### PR TITLE
db: Add simple timeline rendering

### PR DESCRIPTION
**Pull request by V-FEXrt**
Adds a `--simple-timeline` flag similar to `--timeline` with  most of the data removed to generate a much smaller timeline file.

Keeps cmdline, label, job id, start time, and end time. All other info should be pulled from more targeted/filtered timeline files

### CODE REVIEW

File: share/wake/html/timeline_main.js [hunk 1]
```diff
@@ -312,11 +312,6 @@ timeline.on('click', function (properties) {
     }
     let job = jobMap.get(parseInt(properties.item)).job;
     document.getElementById("job").innerHTML = job.job
-    if (job.usage.charAt(8) === '0') {
-        document.getElementById("job").style.color = "green";
-    } else {
-        document.getElementById("job").style.color = "red";
-    }
```
Comment (V-FEXrt) [inline share/wake/html/timeline_main.js:319]: Its unclear to me if anyone is depending on the coloring here but its not super color blind friendly and removing it simplifies the diff for this PR

File: tools/wake/describe.cpp [hunk 2]
```diff
@@ -312,6 +312,42 @@ void describe_timeline(const std::vector<JobReflection> &jobs,
          "</html>\n";
 }
 
+void describe_simple_timeline(const std::vector<JobReflection> &jobs,
```
Comment (V-FEXrt) [inline tools/wake/describe.cpp:315]: should probably make a shared function to DRY it up here but each rendering feels subtly different in such a way that following a DRY path here feels incorrect.

File: tools/wake/main.cpp [hunk 3]
```diff
@@ -281,6 +285,7 @@ void print_help(const char *argv0) {
     << "    --failed   -f    Capture jobs which failed last build"                       << std::endl
     << "    --tag    KEY=VAL Capture jobs which are tagged, matching KEY and VAL globs"  << std::endl
     << "    --timeline       Report timeline of captured jobs as HTML"                   << std::endl
+    << "    --simple-timeline       Report timeline of captured jobs as HTML"            << std::endl
```
Comment (V-FEXrt) [inline tools/wake/main.cpp:288]: I need to fix the copy and spacing here. Will do that first thing tomorrow

File: src/runtime/database.cpp [hunk 4]
```diff
@@ -1177,6 +1177,24 @@ std::string Time::as_string() const {
...
+JAST JobReflection::to_simple_json() const {
+  JAST json(JSON_OBJECT);
+  json.add("job", job);
+  json.add("label", label.c_str());
+
+  std::stringstream commandline_stream;
+  for (const std::string &line : commandline) {
+    commandline_stream << line << " ";
+  }
+  json.add("commandline", commandline_stream.str());
```
Comment (colinschmidt) [inline src/runtime/database.cpp:1189]: I think we should include `tags` to more closely mirror the information captured in `--simple`

**Review by colinschmidt — APPROVED** [review]
LGTM

**Pull request merged**